source/_integrations/history_stats.markdown

```diff
@@ -129,6 +129,11 @@ duration:
 
 </div>
 
+### Video Tutorial
+This video tutorial explains how you can use history stats and how you can create a daily barchart graph to track occupance of a room or how long the lights in a room a on a daily basis.
```

Comment:
```suggestion
This video tutorial explains how you can use history stats. It also shows how you can create a daily bar chart graph to visualize things such as occupancy, or how long the lights are on in a particular room.
```
Because I find short sentences easier to read, I took some liberties here. I also removed the *on a daily basis*, since daily is already mentioned in the daily bar chart. 
Let me know what you think.

Reply:
Yes sure, no problem of course!